src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -22,19 +22,25 @@ partial class mincore
         [DllImport(Libraries.CoreFile_L1, EntryPoint = "GetFullPathNameW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
         private static extern int GetFullPathNamePrivate(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero);
 
-        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero)
+        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer)
```

Comment:
It's not aesthetics.  If we switch to something like MCG which generates these interop functions for us, we'll delete these ```Interop.*.cs``` files, and there goes your special handling.  If someone looks at a function named Zoo in these Interop files, they should be able to assume that Zoo behaves identically to Zoo as defined by the platform, as these functions are meant to be EXACTLY what the behavior of the platform, simply making the platform functionality available to C#; these behavior changes nullify that.